modules/optimization/doc/content/source/dirackernels/ReporterTimePointSource.md

```diff
@@ -0,0 +1,22 @@
+# ReporterTimePointSource
+
+!syntax parameters /DiracKernels/ReporterTimePointSource
+
+## Overview
+
+A `ReporterTimePointSource` reads in multiple point sources from a [Reporter](Reporters/index.md) or [VectorPostprocessor](VectorPostprocessors/index.md).  The point source values and coordinates are updated as the values are changed.
```

Comment:
we need more details on how the time dependence plays out

the points are moving in time right?
and the values are changing too?
how are the vectors indexed with time?